src/components/base/outline-accordion/outline-accordion.css

```
@@ -0,0 +1,14 @@
+.accordion-buttton {
```

```suggestion
.accordion-button {
```

Nitpick: `buttton` should be spelled with two `t`s.

---

Not nitpick at all That's me being sloppy dev. Good catch!